src/cli/net/server/mod.rs

```diff
@@ -21,23 +26,69 @@ pub mod handlers;
 pub enum MpcServerError {
     #[error(transparent)]
     BadQueryString(#[from] QueryRejection),
+    #[error("header not found: {0}")]
+    MissingHeader(String),
+    #[error("invalid header: {0}")]
+    InvalidHeader(BoxError),
+    #[error(transparent)]
+    BadPathString(#[from] PathRejection),
+    #[error(transparent)]
+    HttpError(#[from] hyper::Error),
+    #[error("parse error: {0}")]
+    SerdeError(#[from] serde_json::Error),
```

Comment:
is this used anywhere?

Reply:
not anymore. removed

Reply:
sorry, this is also used [here](https://github.com/martinthomson/raw-ipa/pull/94/files#diff-2bb5f35b32641587c1cae31f2b3db2dc028d3606820ce9fda87da4841ec9752cR17)